CMakeLists.txt

```diff
@@ -1,15 +1,58 @@
-cmake_minimum_required(VERSION 3.8.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.9.2 FATAL_ERROR)
+
+if("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" GREATER_EQUAL 3.9.2)
+   # Set and use the newest cmake policies that are available, and have been
+   # verified to work
+  cmake_policy(VERSION 3.13.1) # As of 2018-12-04 GDCM builds with cmake version 3.13.1 new policies.
+endif("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" GREATER_EQUAL 3.9.2)
+
+# GDCM version 3.0.0 will only support C++11 and greater
+if(CMAKE_CXX_STANDARD EQUAL "98" )
+   message(FATAL_ERROR "CMAKE_CXX_STANDARD:STRING=98 is not supported in ITK version 5 and greater.")
```

Comment:
Copy-paste mistake: we should not mention ITK here.

Reply:
Fixed.  Thanks.